bob/src/cluster/support_types.rs

```diff
@@ -25,6 +25,38 @@ impl<T: Eq + core::hash::Hash> HashSetExt<T> for HashSet<T> {
     }
 }
 
+#[derive(Debug)]
+pub(crate) struct RemotePutResponse {
+    affected_replicas: usize
+}
+
+impl RemotePutResponse {
+    pub(super) fn new(affected_replicas: usize) -> Self {
+        Self { affected_replicas }
+    }
+
+    pub(super) fn affected_replicas(&self) -> usize {
+        self.affected_replicas
+    }
+}
+
+#[derive(Debug)]
+pub(crate) struct RemotePutError {
+    affected_replicas: usize,
+    #[allow(dead_code)] // We need error for debug print
+    error: Error
```

Comment:
Add also a function, that returns this error. It would be better to make it visible, that error can be accessed outside

Reply:
Another statement with allowed dead code? :) Why not defer it until we actually need this error?

Reply:
Only function will have `#[allow(dead_code)]`. The field in this case is not needed to be marked with this attribute

Reply:
Actually it makes no difference here. So you can do it according to your own preference